platforms-maturity-model/v1/index.md

```diff
@@ -0,0 +1,482 @@
+
+As an organization learns and evolves, so will its platform practices mature and develop. However, an organization should not strive for "Level 4" maturity "just because" - instead, each organization should evaluate its own particular requirements and invest accordingly in the platform capabilities that are most appropriate for its needs. Said another way, this model is not a ruler by which the success for any given platform, person, team, or organization should be measured. It is instead offered as a tool for users to identify and articulate opportunities for enhancement and to solicit support to optimize software delivery and platform practices.
+The most important piece of any content is the context it was written in. The following sections lay out some context behind the model as well as some expectations for you, the reader.
+
+### Target Audience
+
+Each reader brings a unique context and will take unique learnings from this model. Following are some personas we have in mind, along with their possible motivations for engaging with this model:
+
+* **CTOs, VPs and directors of technology**: Leaders looking to map a path to digital transformation and greater developer productivity
+* **Engineering managers**: Groups and individuals seeking to empower engineers to provide value with less overhead and higher efficiency
+* **Enterprise architects**: Individuals navigating the modern technology landscape who seek a value- and solution-oriented perspective on technology problems
+* **Platform engineers and platform product managers**: Teams and people seeking to build the best possible experience for platform builders and platform users
+* **Product vendors and project maintainers**: Organizations and engineers wishing to design tools and deliver messages to enable users to succeed with platforms and capabilities
+* **Application and product developers**: Platform users seeking to understand in more detail what they might expect of an internal platform
+
+
+### What are platforms and platform engineering?
+
+The use of this model to evaluate organizational maturity and opportunities for enhancement depends on a common understanding of its aspects. Please read the [CNCF Platforms Definition white paper](https://tag-app-delivery.cncf.io/whitepapers/platforms/) first for complete context.
+
+**Platform engineering** encompasses all parts of building platforms and their capabilities - the people, processes, policy, technology, and business outcomes that drive the need for, and delivery of, internal platforms.
+
+More specifically, **platforms** are the implementation of a foundation on which to build digital products. Their goals are to provide consistency and efficiency for software development and operations. Platform engineering refers to the greater practice of planning and providing the necessary platforms for organizations by people who specialize in their capabilities and the user experiences surrounding them.
```

Comment:
We redefine Platform engineering in this paragraph although it has already been defined at least to some extent above in line 42. This might confuse the reader as we jump between PE and P back and forth. Not super sure how to dissolve this right now, but I'd suggest starting with the definition of Platforms and then leading over to Platform engineering merging the two slightly differing definitions in 42 and 44. WDYT?

Reply:
Agree with @puja108's point, we define "platform engineering" slightly differently in each paragraph. Also the definition of platforms is a bit ambiguous and different than in the Definition paper. As @puja108 suggests perhaps we can combine the two paragraphs like this:

"_Platforms_ aggregate capabilities needed for development and operation of digital products and applications. They provide consistency and efficiency for software development and delivery. _Platform engineering_ is the practice of planning and providing such computing platforms to developers and users and encompasses all parts of planning and providing platforms and their capabilities - the people, processes, policy, technology, and business outcomes that define the need for and delivery of internal platforms."

Reply:
👍 Thanks both! I have pulled in the new paragraph @joshgav just with links back to the glossary for the two key terms 🙌